webui/tests/selenium/webui-selenium-test.py

```diff
@@ -330,6 +312,16 @@ def logout(self):
 
 # Methods used for waiting and clicking
 
+    def getChromedriverpath(self):
+        # On OS X: Chromedriver path is 'usr/local/lib/chromium-browser/chromedriver'
+        for chromedriverpath in {'/usr/lib/chromium-browser/chromedriver', '/usr/local/lib/chromium-browser/chromedriver'}:
+            try:
+                os.path.isfile(chromedriverpath)
+            except FileNotFoundError:
+                pass
+            else:
+                return chromedriverpath
+
```

Comment:
This new method searches for the right path where the chromedriver is located. On os x, `/usr/lib/` is reserved for system access only. `/usr/local/lib` is the right path.